benefits/enrollment/templates/enrollment/index.html

```diff
@@ -1,24 +1,17 @@
-{% extends "core/page.html" %}
+{% extends "core/base.html" %}
 
-{% block main_content %}
-  {% include "core/includes/sign-out-link.html" %}
-  {{ block.super }}
-{% endblock main_content %}
+{% block nav-buttons %}
+  <div class="d-flex justify-content-end">{% include "core/includes/sign-out-link.html" %}</div>
+{% endblock nav-buttons %}
 
-{% block icon_title %}
-  {{ block.super }}
-{% endblock icon_title %}
+{% block inner-content %}
+  <div class="col-12 col-sm-12 col-lg-10">
+    <h2 class="media-title"></h2>
```

Comment:
Empty `<h2>`?

Reply:
I meant to start a discussion on this, but you beat me to it!

This is here temporarily because the spacing between the media-list and whatever is above it is defined on `media-title`. This media-list on this page does not have a media-title though. Does @machikoyasuda already have an issue that covers this? Does #1028 actually cover this?

Reply:
Yes I believe this is handled by one of the cases in #1028: `h1 + .media-list`

Reply:
Ok, I removed the empty `<h2>` in 2ecf25a22597d5ed07a7bde1c836aa7fa99a5d52 and am good with #1028 fixing the spacing for this page, which now looks like this:

<img src="https://user-images.githubusercontent.com/25497886/195220128-e3792f9d-0ca0-4e76-bb8e-a0accadfa405.png" width="450">